frontend/src/components/form/api/__tests__/ApiColorPicker.spec.js

```diff
@@ -52,20 +52,18 @@ describe('An ApiColorPicker', () => {
     await user.click(screen.getByLabelText('Speichern'))
 
     // then
-    await waitFor(async () => {
-      const inputField = await screen.findByLabelText(FIELD_LABEL)
-      expect(inputField.value).toBe(COLOR_2)
-      expect(apiMock.getMocks().patch).toBeCalledTimes(1)
-    })
+    const inputField = await screen.findByLabelText(FIELD_LABEL)
```

Comment:
are you sure that you can remove the waitFor?

Reply:
Yes. It still works and now we don't have the feedback-loop anymore